tests/test_column_level_knowledge_propagator.py

```diff
@@ -161,12 +172,22 @@ def test_update_undocumented_columns_with_prior_knowledge():
     assert (
         yaml_file_model_section["columns"][0]["description"] == "THIS COLUMN IS UPDATED FOR TESTING"
     )
-    assert yaml_file_model_section["columns"][0]["meta"] == {"my_key": "my_value"}
-    assert set(yaml_file_model_section["columns"][0]["tags"]) == set(["my_tag1", "my_tag2"])
+    assert yaml_file_model_section["columns"][0]["meta"] == {
+        "my_key": "my_value",
+        "my_new_key": "my_new_value",
+    }
+    assert set(yaml_file_model_section["columns"][0]["tags"]) == set(
+        ["my_tag1", "my_tag2", "my_tag3", "my_tag4"]
+    )
 
     assert target_node.columns["customer_id"].description == "THIS COLUMN IS UPDATED FOR TESTING"
-    assert target_node.columns["customer_id"].meta == {"my_key": "my_value"}
-    assert set(target_node.columns["customer_id"].tags) == set(["my_tag1", "my_tag2"])
+    assert target_node.columns["customer_id"].meta == {
+        "my_key": "my_value",
+        "my_new_key": "my_new_value",
+    }
+    assert set(target_node.columns["customer_id"].tags) == set(
```

Comment:
Changed existing behavior: `tags` and `meta` for progenitor were taken into account, but when dbt-osmosis was run, `tags` and `meta` assigned to the target node itself disappeared.